packages/components/src/button/index.js

```diff
@@ -104,6 +187,13 @@ export function Button( props, ref ) {
 
 	const element = (
 		<Tag
+			css={ css`
+				${ getBaseStyles( props ) }
+				${ getBusyStyles( props ) }
+				${ getSmallStyles( props, hasIcon, hasText ) }
+				${ getPressedStyles( props ) }
+				${ getIconStyles( hasIcon, hasText ) }
```

Comment:
Something I don't like about this approach is that there is a non-obvious order requirement to some of these. For example, if you move the Icon styles above the Pressed styles, you'll get padding conflicts. I _think_ this is sort of inherent to the `css` prop itself and, alas, it might just be something that needs to be swallowed, but maybe someone has ideas for cleaner ways to do this.